build-system/common/update-design-review-issues.js

```diff
@@ -474,6 +468,11 @@ async function closeStaleIssues(token, repo, issuesWithSessionDate) {
   // Compensate duration so that we swap only once the session has ended.
   now.setHours(now.getHours() - sessionDurationHours);
 
+  // We may run matching a session's end by the minute, this prevents off-by-one.
+  now.setSeconds(Math.max(1, now.getSeconds()));
+
+  console./*OK*/ log('Session close cutoff:', now);
```

Comment:
Intentional?

Reply:
Yep. It's useful to see during real runs.